config/jobs/image-pushing/k8s-staging-provider-azure.yaml

```diff
@@ -0,0 +1,40 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-sigs/cloud-provider-azure:
+    # The name should be changed to match the repo name above
+    - name: post-provider-azure-push-images
+      cluster: test-infra-trusted
+      annotations:
+        # This is the name of some testgrid dashboard to report to.
+        # If this is the first one for your sig, you may need to create one
+        testgrid-dashboards: sig-cloud-provider-image-pushes
```

Comment:
Refer here https://github.com/kubernetes/test-infra/blob/9bcc9e7640e70cbcaa40714b88a8cf4a1e3f4d29/config/testgrids/kubernetes/sig-cloud-provider/azure/config.yaml#L1 to add a dashboard